docker/linux-image.sh

```diff
@@ -58,27 +115,26 @@ main() {
             # there is no stable port
             arch=ppc64
             # https://packages.debian.org/en/sid/linux-image-powerpc64
-            kversion='5.*'
-            kernel="${kversion}-powerpc64"
             libgcc="libgcc-s1"
+            kernel='5.*-powerpc64'
             debsource="deb http://ftp.ports.debian.org/debian-ports unstable main"
             debsource="${debsource}\ndeb http://ftp.ports.debian.org/debian-ports unreleased main"
             # sid version of dropbear requires these dependencies
             deps=(libcrypt1:"${arch}")
             ;;
         powerpc64le)
             arch=ppc64el
-            kernel="${kversion}-powerpc64le"
+            kernel="5.*-powerpc64le"
             ;;
         s390x)
             arch=s390x
-            kernel="${kversion}-s390x"
+            kernel="5.*-s390x"
             ;;
         sparc64)
             # there is no stable port
             # https://packages.debian.org/en/sid/linux-image-sparc64
-            kernel='*-sparc64'
-            libgcc="libgcc-s1"
+            # the latest image doesn't work as of 5.16.18-1
+            kernel='5.16.0-6-sparc64'
```

Comment:
```suggestion
            kernel='5.*-sparc64'
```
The latest image is now [15.17.`](https://packages.debian.org/sid/linux-image-5.17.0-1-sparc64), which has all the needed files. (This change should resolve the build failure)

Reply:
Let me test, because a wildcard last time led to an empty package being used without the proper files even though I was worried this exact issue would happen.

Reply:
Yeah 5.17.6 doesn't work: it's a dummy [package](https://packages.debian.org/en/sid/sparc64/linux-image-sparc64/filelist). We should be able to get away with [5.17.2](https://packages.debian.org/sid/sparc64/linux-image-5.17.0-2-sparc64/filelist) however, but this is a very temporary fix.

```text
/usr/share/bug/linux-image-sparc64/presubj
/usr/share/doc/linux-image-sparc64
```